modules/infotext_utils.py

```diff
@@ -230,7 +230,7 @@ def restore_old_hires_fix_params(res):
     res['Hires resize-2'] = height
 
 
-def parse_generation_parameters(x: str):
+def parse_generation_parameters(x: str, skip_fields: list[str] | None = None):

```

Comment:
no that would cause issues
if someone is to potentially use this function with a customizable `skip_fields` list object (which can potentially be an empty list)
it would be indistinguishable to not passing the arg